### ISSUE TITLE
Terminal accessibility menu keybinding not read with screen reader attached 

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#172199]
Testing #171862

With editor.accessibilitySupport enabled and Windows Narrator running, when I click into the terminal, Narrator reads the terminal number, type (pwsh), and keybindings to navigate within the terminal history, but does not read the keybinding for Show Terminal Accessibility Help. Shortcuts are unchanged from defaults:

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/30305945/214355855-e82d33b7-2d0e-4639-be84-d412633fa386.png">

**Comment by Tyriar** [comment]
I see a couple of problems here:

https://github.com/microsoft/vscode/blob/c35bf3b3401cef46d2c4dee464adb52c97b81228/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts#L1999-L2009

- We append to `label` without a space, so it ends up as "Terminal {0}Press ...`
- It only shows when `screenReaderOptimized`, but it should always show (and include instructions on how to turn it on)

**Comment by meganrogge** [comment]
Additionally, I can repro it not being included in the alt text even when `screenReaderOptimized` which is surprising when looking at the code. I do see the massage about using the up and down to navigate
